desk

```diff
@@ -91,6 +92,11 @@ cmd_init() {
 
 cmd_go() {
     local TODESK="$1"
+    if [ -z "$TODESK" -a -f "${PWD}/.desk" ]; then
+	# if no desk is specified try the .desk file in the current directory
+	read TODESK < "${PWD}/.desk"
```

Comment:
I'd prefer we go with `Deskfile` here (resembles `Makefile`, `Dockerfile`, etc.).